charts/partners/fortanix/fortanix-secrets-injector/OWNERS

```diff
@@ -0,0 +1,11 @@
+chart:
```

Comment:
This is not going to work, Owners will get pushed automatically as part of the Part of the partner connect process documented here: https://github.com/openshift-helm-charts/charts/tree/main/docs#preparation

Reply:
Thanks David for your comment. We have completed the partner connect process on Redhat portal. Is there an expected wait time for the OWNER update on github. Otherwise will raise a case with Redhat.

Reply:
Yes raising a case will be next step if you don't see that coming in soon. I will contact some folks from our end. But with a case things will move faster.

Reply:
Thanks David. As you have pointed out, OWNERS is now committed with Fortanix details. I have now re-pushed the chart to match the correct chart name and partner folder.